### ISSUE TITLE
[FR] Warn for string & array concatenation

### ISSUE THREAD

**Issue by kiler129** [kalessil/phpinspectionsea#1126]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Feature request                          |
| Plugin         | Php Inspections (EA Extended) v.3.0.8.1 |
| Language level | PHP 7.2                        |

#### Current behavior:
The code:
`$this->logger->error('Invalid country code "{code}", assuming US'. ['code' => $country]);`
does not produce any warnings.

#### Expected behavior
Notification should be displayed, since running this code will cause `E_NOTICE` about array to string conversion. In the exact case described above it was just a typo (`.` instead of `,`).

#### Environment details
```
PhpStorm 2018.3
Build #PS-183.4284.150, built on November 21, 2018
Licensed to [redacted]
Subscription is active until [redacted]
JRE: 1.8.0_152-release-1343-b15 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.15.0-39-generic
```

**Comment by kalessil** [comment]
Done!
